lib/src/models/parsed_output.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:ledger_cardano/src/utils/constants.dart';
+import 'parsed_asset_group.dart';
+import 'parsed_output_destination.dart';
+import 'parsed_datum.dart';
+
+part 'parsed_output.freezed.dart';
+
+@freezed
+class ParsedOutput with _$ParsedOutput {
+  const ParsedOutput._();
+
+  const factory ParsedOutput({
+    required TxOutputFormat format,
+    required String amount,
```

Comment:
Is this a `string` in typescript? All amounts are `BigInt` in cardano. Please change this to BigInt despite of what is in typescript.

Reply:
Whenever you have some `amount`, you should use BigInt.

Reply:
Still needs doing